src/staking_rewards/data.csv

```diff
@@ -350,97 +350,4 @@ Until epoch,Estimated date,Reward per epoch %,Estimated annualized rewards %
-32298,2024-05-21T11:24:55.921Z,0.000229,2.0779
-32394,2024-05-25T09:03:18.763Z,0.000228,2.0687
-32490,2024-05-29T06:41:41.604Z,0.000227,2.0596
-35514,2024-09-29T04:20:41.108Z,0.000226,2.0504
-35562,2024-10-01T03:09:52.529Z,0.000225,2.0412
-35586,2024-10-02T02:34:28.239Z,0.000224,2.0321
-35610,2024-10-03T01:59:03.950Z,0.000223,2.0229
-35634,2024-10-04T01:23:39.660Z,0.000222,2.0137
-35658,2024-10-05T00:48:15.370Z,0.000221,2.0046
-35682,2024-10-06T00:12:51.081Z,0.000220,1.9954
-35706,2024-10-06T23:37:26.791Z,0.000219,1.9863
-35730,2024-10-07T23:02:02.501Z,0.000218,1.9771
-35754,2024-10-08T22:26:38.212Z,0.000216,1.9588
-35778,2024-10-09T21:51:13.922Z,0.000215,1.9496
-35802,2024-10-10T21:15:49.633Z,0.000213,1.9313
-35826,2024-10-11T20:40:25.343Z,0.000212,1.9222
-35850,2024-10-12T20:05:01.053Z,0.000210,1.9039
-35874,2024-10-13T19:29:36.764Z,0.000208,1.8856
-35898,2024-10-14T18:54:12.474Z,0.000205,1.8581
-35922,2024-10-15T18:18:48.184Z,0.000203,1.8398
-35946,2024-10-16T17:43:23.895Z,0.000200,1.8124
-35970,2024-10-17T17:07:59.605Z,0.000196,1.7758
-35994,2024-10-18T16:32:35.315Z,0.000192,1.7393
-36018,2024-10-19T15:57:11.026Z,0.000188,1.7027
-36042,2024-10-20T15:21:46.736Z,0.000183,1.6571
-36066,2024-10-21T14:46:22.446Z,0.000177,1.6023
-36090,2024-10-22T14:10:58.157Z,0.000171,1.5476
-36114,2024-10-23T13:35:33.867Z,0.000164,1.4838
-36138,2024-10-24T13:00:09.577Z,0.000155,1.4018
-36162,2024-10-25T12:24:45.288Z,0.000146,1.3198
-36186,2024-10-26T11:49:20.998Z,0.000136,1.2289
-36210,2024-10-27T11:13:56.708Z,0.000125,1.1289
-36234,2024-10-28T10:38:32.419Z,0.000114,1.0291
-36258,2024-10-29T10:03:08.129Z,0.000102,0.9203
-36282,2024-10-30T09:27:43.840Z,0.000091,0.8206
-36306,2024-10-31T08:52:19.550Z,0.000081,0.7301
-36330,2024-11-01T08:16:55.260Z,0.000072,0.6487
-36354,2024-11-02T07:41:30.971Z,0.000063,0.5674
-36378,2024-11-03T07:06:06.681Z,0.000056,0.5042
-36402,2024-11-04T06:30:42.391Z,0.000049,0.4410
-36426,2024-11-05T05:55:18.102Z,0.000044,0.3959
-36450,2024-11-06T05:19:53.812Z,0.000039,0.3509
-36474,2024-11-07T04:44:29.522Z,0.000034,0.3058
-36498,2024-11-08T04:09:05.233Z,0.000030,0.2698
-36522,2024-11-09T03:33:40.943Z,0.000027,0.2428
-36546,2024-11-10T02:58:16.653Z,0.000024,0.2158
-36570,2024-11-11T02:22:52.364Z,0.000021,0.1888
-36594,2024-11-12T01:47:28.074Z,0.000019,0.1708
-36618,2024-11-13T01:12:03.784Z,0.000017,0.1528
-36642,2024-11-14T00:36:39.495Z,0.000015,0.1348
-36666,2024-11-15T00:01:15.205Z,0.000013,0.1168
-36690,2024-11-15T23:25:50.915Z,0.000011,0.0988
-36714,2024-11-16T22:50:26.626Z,0.000010,0.0898
-36738,2024-11-17T22:15:02.336Z,0.000008,0.0719
-36762,2024-11-18T21:39:38.047Z,0.000007,0.0629
-36786,2024-11-19T21:04:13.757Z,0.000006,0.0539
-36810,2024-11-20T20:28:49.467Z,0.000005,0.0449
-36834,2024-11-21T19:53:25.178Z,0.000004,0.0359
-36858,2024-11-22T19:18:00.888Z,0.000003,0.0269
-36882,2024-11-23T18:42:36.598Z,0.000002,0.0180
-36930,2024-11-25T17:31:48.019Z,0.000001,0.0090
+81662,2030-01-01T00:00Z,0.000283,2.5741
```

Comment:
data.json says until epoch 90000, not 81662

Reply:
I thought it would look nicer if the graph ended in 2030 rather than on a random date. Do the JSON and csv need to match?

Reply:
That doesn't make sense. CSV doesn't affect the graph

Reply:
Either way, don't manipulate the data, especially the data that users download and analyze and make own graphs out of. IF we want to display less data then we should change that in the display component :shrug:

and that can be a separate pullrequest if tadej insists on not letting users to zoom where they want to

Reply:
Yeah, this makes sense. I changed the CSV to include epoch 90000. We can create a new pr for the display component if needed. I also think it is nicer to show all the data and then can users to zoom in if they want.